File: Java/MathematicalPrograms/2DShapes/AreaAndPerimeterOfParallelogram.java
import java.util.Scanner;
public class AreaAndPerimeterOfParallelogram {
	public static void main(String[] args) {
		Scanner myObj = new Scanner(System.in);
		System.out.println("Enter the length of the Parallelogram");
		int length = myObj.nextInt();
		System.out.println("Enter the breadth of the Parallelogram");
		int breadth = myObj.nextInt();
		double area = length*breadth;
		double perimeter = 2*(length+breadth);
		System.out.println("Area of the Parallelogram is " + area);
		System.out.println("Perimeter of the Parallelogram is " + perimeter);
	}
}

File: C++/MathematicalPrograms/2DShapes/AreaAndPerimeterOfParallelogram.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
	int Length, Breadth;
	double area, perimeter;
	cout << "Enter a Length measure";
	cin >> Length;
	cout << "Enter a Breadth measure";
	cin >> Breadth;
	area = Length*Breadth;
	perimeter = 2*(Length+Breadth);
	cout << "Area of the Parallelogram is " << area;
	cout << "\nPerimeter of the Parallelogram is " << perimeter;
}